cmd/machine-config-controller/start.go

```diff
@@ -210,7 +210,7 @@ func createControllers(ctx *ctrlcommon.ControllerContext) []ctrlcommon.Controlle
 			ctx.InformerFactory.Machineconfiguration().V1().MachineConfigPools(),
 			ctx.KubeInformerFactory.Core().V1().Nodes(),
 			ctx.KubeInformerFactory.Core().V1().Pods(),
-			ctx.InformerFactory.Machineconfiguration().V1alpha1().MachineOSBuilds(),
+			ctx,
```

Comment:
Any reason why we can't just pass the mosc informer here instead of the whole context?